jujugui/static/gui/src/app/views/utils.js

```diff
@@ -166,9 +166,10 @@ viewsUtils.formatConstraints = constraints => {
   }, []).join(' ');
 };
 
-if (module && module.exports) {
+try {
```

Comment:
For some reason this died once jujulib was imported (I guess jujulib was setting up a module variable.